test_scripts/API/SetAudioStreamingIndicator/037_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_media_app_RPC_disallowed_by_user.lua

```diff
@@ -0,0 +1,245 @@
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_trigger_getting_device_consent()
+  testCasesForPolicyTable:trigger_getting_device_consent(self, config.application1.registerAppInterfaceParams.appName, config.deviceMAC)
+end
+
+function Test:Precondition_PTU_appPermissionsConsentNeeded_true()
+  local RequestIdGetURLS = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+  EXPECT_HMIRESPONSE(RequestIdGetURLS,{result = {code = 0, method = "SDL.GetURLS", urls = {{url = "http://policies.telematics.ford.com/api/policies"}}}})
+  :Do(function(_,_)
+    self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest", { requestType = "PROPRIETARY", fileName = "filename"})
+
+    EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "PROPRIETARY" })
+    :Do(function(_,_)
+      self.mobileSession:SendRPC("SystemRequest", { fileName = "PolicyTableUpdate", requestType = "PROPRIETARY"}, "SetAudioStreamingIndicator_group1.json")
+
+      EXPECT_HMICALL("BasicCommunication.SystemRequest")
+      :Do(function(_,data)
+        self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate", { policyfile = "/tmp/fs/mp/images/ivsu_cache/PolicyTableUpdate"})
+
+        local function to_run()
+          self.hmiConnection:SendResponse(data.id,"BasicCommunication.SystemRequest", "SUCCESS", {})
+        end
+        RUN_AFTER(to_run, 500)
+      end)
+
+      EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATING"}, {status = "UP_TO_DATE"}):Times(2)
+      :Do(function(_,data)
+        if(data.params.status == "UP_TO_DATE") then
+          EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", {appID = self.applications[config.application1.registerAppInterfaceParams.appName], appPermissionsConsentNeeded = true })
+          :Do(function()
+            local RequestIdListOfPermissions = self.hmiConnection:SendRequest("SDL.GetListOfPermissions", { appID = self.applications[config.application1.registerAppInterfaceParams.appName] })
+
+            EXPECT_HMIRESPONSE(RequestIdListOfPermissions)
+            :Do(function(_,data1)
+              local groups = {}
+              if #data1.result.allowedFunctions > 0 then
+                for i = 1, #data1.result.allowedFunctions do
+                  groups[i] = {
+                    name = data1.result.allowedFunctions[i].name,
+                    id = data1.result.allowedFunctions[i].id,
+                    allowed = true}
+                end
+              end
+              self.hmiConnection:SendNotification("SDL.OnAppPermissionConsent", { appID = self.applications[config.application1.registerAppInterfaceParams.appName], consentedFunctions = groups, source = "GUI"})
+              EXPECT_NOTIFICATION("OnPermissionsChange")
+            end)
+          end)
+        end
+      end)
+    end)
+  end)
+end
+
```

Comment:
@dboltovskyi, I think that in scope of test verification of DISALLOWED -> USER_DISALLOWED is enough, because application changes only allowed functionality and application doesn't change the hmi level. The verification is enough to check requirement.